src/ContactList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Contact from './Contact';
+
+// Converted from a class based component to a functional component
+const ContactList = props => {
+  return (
+    <ul className="contact-list">
+      {props.noContacts}
```

Comment:
We actually don't need to pass a prop `props.noContacts`, if we want to do something in the component to calculate this we can do it passed on the `props.contacts` array that it receives.

Not the end of the world, but just a thought!